api/WebSocket.json

```diff
@@ -23,28 +23,17 @@
               "version_removed": "11",
               "prefix": "Moz",
               "notes": "Message size limited to 16 MB (see <a href='https://bugzil.la/711205'>bug 711205</a>)."
-            },
-            {
-              "version_added": "4",
-              "version_removed": "6",
-              "notes": "Message size limited to 16 MB (see <a href='https://bugzil.la/711205'>bug 711205</a>)."
             }
           ],
           "firefox_android": [
             {
-              "version_added": "14",
-              "notes": "See <a href='https://bugzil.la/695635'>bug 695635</a>."
```

Comment:
I made this edit. https://bugzil.la/695635 is the unprefixed bug and could have been linked in the prefixed entry, but is really just unnessary to link anywhere, and was left over in https://github.com/mdn/browser-compat-data/pull/6608.